internal/objects/objects.go

```diff
@@ -161,6 +164,30 @@ func NewKubeletConfigAutoresizeControlPlane() *machineconfigv1.KubeletConfig {
 	return ctrlPlaneKc
 }
 
+func NewKubeletConfigConfigMap(name string, labels map[string]string, config *machineconfigv1.KubeletConfig) *corev1.ConfigMap {
+	yamlSerializer := serializer.NewSerializerWithOptions(
+		serializer.DefaultMetaFactory, scheme.Scheme, scheme.Scheme,
+		serializer.SerializerOptions{Yaml: true, Pretty: true, Strict: true})
+
+	buff := &bytes.Buffer{}
+	// supervised testing environment
+	// no need to check for error
+	_ = yamlSerializer.Encode(config, buff)
```

Comment:
seem myself good intentioned things like this blow up more often that I like, so let's panic badly if `err != nil`, it should never happen anyway, right? :)

Reply:
Done. Thanks!